From 65220709a1ad9607f99f8f32d48883c0cb9bdf3c Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 3 Jun 2019 18:53:08 -0600
Subject: [PATCH 5/5] [SQUASH][DNM] Merge tag 'android-9.0.0_r40' into
 staging/lineage-16.0_merge-android-9.0.0_r40

Android 9.0.0 release 40

* tag 'android-9.0.0_r40':
  Version bump to PQ3A.190605.003 [core/build_id.mk]
  Version bump to PQ3A.190605.002
  Update Security String to 2019-06-05 Bug:129374896 Merged-In: 2c26c3d4c0314f8a3f905e5a7081556f7b4b353a (cherry picked from commit 3ae4e858b7c169b19a4994dd6b124a68fa234ff0)
  [DO NOT MERGE] Update Security String to 2019-06-01 Bug:129374896 (cherry picked from commit 2db03695f4fccc312947155ae724f48885e472c5)

https://github.com/haggertk/android_build/tree/lineage-16.0_merge-android-9.0.0_r40

Change-Id: If2089879676ef3a333bc1de29a82ffac86dda784
---
 core/build_id.mk         | 2 +-
 core/version_defaults.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 4c46c5cf1..6d0e827d7 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=PQ3A.190505.002
+export BUILD_ID=PQ3A.190605.003
diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 80e5007ea..07e6c95e9 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -231,7 +231,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2019-05-05
+      PLATFORM_SECURITY_PATCH := 2019-06-05
 endif
 
 ifndef PLATFORM_SECURITY_PATCH_TIMESTAMP
-- 
2.17.1

